components/icons/icon-styles.js

```diff
@@ -28,10 +28,9 @@ export const iconStyles = css`
 		pointer-events: none;
 		width: 100%;
 	}
-	:host([dir="rtl"]) svg[mirror-in-rtl],
-	:host([dir="rtl"]) ::slotted(svg[mirror-in-rtl]) {
-		-webkit-transform: scale(-1, 1);
-		transform: scale(-1, 1);
+	svg[mirror-in-rtl],
+	::slotted(svg[mirror-in-rtl]) {
+		transform: var(--d2l-mirror-transform, ${document.dir === 'rtl' ? css`scale(-1, 1)` : css`none`}); /* stylelint-disable-line @stylistic/string-quotes, @stylistic/function-whitespace-after */
```

Comment:
This wasn't what I was expecting... if we need to put the fallback in there anyways, why not just use that and skip the variable? Because that's more expensive?

Reply:
The fallback is static and doesn't get updated when the language of the page changes dynamically. Which doesn't really happen outside our demo pages today, but that doesn't mean it's not kind of nice.

Reply:
That's right yep. The other case that relies on the variable is visual-diff pages.

Reply:
Ah ok, gotcha. Just a bit unfortunate then that the code still needs to know the variable value for the fallback, like both would need to be updated if we changed anything.